api/src/handlers/server_api.rs

```diff
@@ -54,7 +55,12 @@ impl StatusHandler {
 		let (api_sync_status, api_sync_info) = sync_status_to_api(sync_status);
 		Ok(Status::from_tip_and_peers(
 			head,
-			w(&self.peers)?.peer_count(),
+			w(&self.peers)?
+				.peers_iter()
+				.connected()
+				.count()
+				.try_into()
+				.unwrap(),
```

Comment:
Unwraps always make me uneasy, unless we can prove they never reach the bad path. In this case we can't, so could we replace it with an `unwrap_or(0)`?